_includes/v23.1/sql/select-for-update-limitations.md

```diff
@@ -0,0 +1,10 @@
+Locks acquired using {% if page.name == "select-for-update.md" %} `SELECT ... FOR UPDATE` {% else %} [`SELECT ... FOR UPDATE`](select-for-update.html) {% endif %} are lost on [lease transfers](architecture/replication-layer.html#epoch-based-leases-table-data) and [range splits and merges](architecture/distribution-layer.html#range-merges). This happens because currently `SELECT ... FOR UPDATE` is implemented using [unreplicated locks](architecture/transaction-layer.html#unreplicated-locks).
+
+If a lease transfer or range split/merge occurs on a range held by an unreplicated lock, the lock is lost, and the following behaviors can occur:
```

Comment:
"... on a range with an unreplicated lock".

Also FWIW, I'm not sure if the term "lost" or "dropped" is better here?

Reply:
I prefer "dropped" thanks for the suggestion. "Loss" might imply something about correctness that is inappropriate.

I've actually gone ahead and changed all uses of "lose/loss" with "dropped" in this PR